ebmlite/core.py

```diff
@@ -1295,15 +1244,13 @@ def _getInt(d, k, default):
 
         return eclass
 
-
     def __repr__(self):
         try:
-            return "<%s %r from '%s'>" % (self.__class__.__name__, self.name,
-                                          self.filename or self.source)
+            return b"<%s %r from '%s'>" % (self.__class__.__name__, self.name,
```

Comment:
I think `__repr__()` is supposed to return a string.

Reply:
good catch, I'll check all the __repr__/__str__ methods